src/cgame/cg_rocket_datasource.cpp

```diff
@@ -32,6 +32,7 @@ Maryland 20850 USA.
 ===========================================================================
 */
 
+#include <cstddef>     // for std::size_t
```

Comment:
Looks like a good point. 

Out of curiosity, is it for reduce compile time (less headers, even system => up compile performance), or just for 'do not touch original quake? sources', or smth else?